server/src/main/java/chat/database/UsersBase.java

```diff
@@ -0,0 +1,69 @@
+package chat.database;
+
+import chat.model.User;
+import org.apache.log4j.Logger;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UsersBase {
+
+    private static final Logger LOGGER = Logger.getLogger(UsersBase.class);
+    private static final String URL = "jdbc:mysql://localhost:3306/Users?serverTimezone=UTC";
+    private static final String NAME_BASE = "root";
+    private static final String PASS_BASE = "root123321";
+    private Connection connection;
+    private PreparedStatement preparedStatement;
+    private static final String SQL_INSERT = "INSERT INTO user (name, password) VALUES (?,?)";
+    private static final String SQL_SHOW = "SELECT * FROM user";
+
+    public UsersBase() throws SQLException, ClassNotFoundException {
+        this.connection = connectionBase();
+    }
+
+    public void addToDataBase(String name, String password) throws SQLException {
```

Comment:
This method adds user so it is better to name it like "saveUser", also it is better if the method will get exact user instead name and password

Reply:
Don't throw an exception, you should catch it. Try to use "Try with resources"

Reply:
++